language/move-core/types/Cargo.toml

```diff
@@ -30,5 +30,6 @@ regex = "1.4.3"
 serde_json = "1.0.64"
 
 [features]
+address20 = []
```

Comment:
Does this add the feature all the time? I'm confused

Reply:
No it just defines the feature. To enable it `address20` needs to be adde to `default` (or explicitly specified when adding the move-core-types dependency)